extended.md

```diff
@@ -472,12 +415,21 @@ Jede Rolle ist wichtig, egal ob Bürger oder Mafia Goon, also verschwendet sie n
 
       **Tipp**: Freimaurer können einfach zusammenarbeiten und sich gegenseitig decken beziehungsweise bestätigen.
 
-      **Schwierigkeitsgrad**: Einfach
-
       **Tipp für den *Spielleiter***: Schreib den Freimaurern einfach so etwas wie “Deine Brüder sind <andere Freimaurer>”
 
       **Variante**: "Offene Sekte": Die Freimaurer bekommen einen Chat, den sie—ähnlich der Mafia—nur nachts verwenden dürfen.
+  
+  - ### **Bürgermeister**
+      {: #buergermeister}
+
+      **Alias**: Doublevoter
+
+      **Fähigkeit**: Der Bürgermeister funktioniert wie eine zweite Rolle. Sie wird von dem Spielleiter zufällig verteilt am Anfang des Spiels. Der Spieler der die Bürgermeister Rolle zusätzlich bekommen hat eine doppelte Stimme in der Wahl des Lynchens.
```

Comment:
"[...] Der Spieler der die [...]" => "[...] Der Spieler**,** der die [...]"
"[...] zusätzlich bekommen hat [sic] eine doppelte Stimme in der Wahl des Lynchens. [...]"